.github/workflows/tag.yaml

```diff
@@ -0,0 +1,28 @@
+name: create_tag
+on: 
+  pull_request:
+    types: [closed]
+
+permissions:
+  contents: write
+
+jobs:
+  create_tag:
+    name: "Create tag"
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release-pr') && startsWith(github.event.pull_request.title, format('chore{0} Prepare', ':'))
+    runs-on: ubuntu-22.04
+    steps:
+    - name: "Set release tag"
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github.event.pull_request.labels.*.name) }}
+      run: |
+        RELEASE_TAG=$(echo "$GITHUB_CONTEXT" | jq '.[1]' | tr -d '"')
```

Comment:
This extracts RELEASE_TAG using jq '.[1]', which assumes that the release label is always at the second position in the array. can you please add some error handling here for if the order of labels changes, this could lead to incorrect tag extraction.

Reply:
Fixed it by fetching tag starting with `v` from the list of labels.